.secrets.baseline

```diff
@@ -206,7 +206,7 @@
         "filename": "tests/io/test_data_catalog.py",
         "hashed_secret": "15dd2c9ccec914f1470b4dccb45789844e49cf70",
         "is_verified": false,
-        "line_number": 529
+        "line_number": 530
```

Comment:
no idea; why this got created.

Reply:
This is automatic, I'm guessing it's because changes were made to `test_data_catalog`.